src/main/java/com/shrralis/ssdemo1/configuration/SecurityConfigAdapter.java

```diff
@@ -52,7 +52,6 @@ protected void configure(HttpSecurity http) throws Exception {
 				.and()
 				.authorizeRequests()
 				.antMatchers("/auth/logout").authenticated()
-				.antMatchers("/admin**/**").access("hasRole('ADMIN')")
```

Comment:
Revert

Reply:
but we don't need this because of `@Secured` above our Controller